server/core/src/https/v1.rs

```diff
@@ -17,14 +17,19 @@ use kanidm_proto::internal::{AppLink, IdentifyUserRequest, IdentifyUserResponse}
 use kanidm_proto::v1::{
     AccountUnixExtend, ApiToken, ApiTokenGenerate, AuthIssueSession, AuthRequest, AuthResponse,
     AuthState as ProtoAuthState, CUIntentToken, CURequest, CUSessionToken, CUStatus, CreateRequest,
-    CredentialStatus, DeleteRequest, Entry as ProtoEntry, GroupUnixExtend, ModifyRequest,
+    CredentialStatus, DeleteRequest, Entry as ProtoEntryReal, GroupUnixExtend, ModifyRequest,
```

Comment:
`utoipa` is will use the exact path/name of the type you specify in `request_body=` which means to say it accepts "ProtoEntry" will require the type to actually have that name in this file.